hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -255,6 +265,22 @@ public void start() {
         TimeUnit.MILLISECONDS);
   }
 
+  private void stopSyncDataFromOMThread() {
+    scheduler.shutdownNow();
+    LOG.debug("Shutdown the OM DB sync scheduler.");
+  }
+
+  @Override
+  public boolean triggerSyncDataFromOMImmediately() {
+    if (!isSyncDataFromOMRunning) {
+      stopSyncDataFromOMThread();
+      scheduler = Executors.newScheduledThreadPool(1);
+      startSyncDataFromOM(0L);
```

Comment:
nit: add a few lines of comments explaining why we need to stop current sync thread and start another one (to reset the timer)

Reply:
Thanks, done.